index.js

```diff
@@ -14,7 +22,18 @@ function hide(args) {
 // show elements
 function show(args) {
```

Comment:
- [ ] The project details popup is not displaying correctly on the desktop version. We actually want our design to closely resemble the provided [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).  The background is overflowing the main container of the popup overlay. Kindly ensure to improve upon this aspect.

**Your popup overly display:**
![image](https://user-images.githubusercontent.com/12745474/131717340-56dacafa-c44f-4bef-a6b2-b520203e9c11.png)

**The original popup overly display:**
![image](https://user-images.githubusercontent.com/12745474/131717995-1a977527-bf39-4e32-994c-aa5b5f24a4ae.png)